.env.dist

```diff
@@ -22,14 +22,14 @@ NATS_ENVIRONMENT_NAME=
 # ENS
 CHAIN_ID=73799
 CHAIN_NAME=volta
-ENS_URL=https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/
+ENS_URL=https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org
```

Comment:
I beleive production environment corresponds to EWC. So here should be private EWC URL